package.json

```diff
@@ -30,6 +30,7 @@
   },
   "dependencies": {
     "@radix-ui/react-checkbox": "1.0.4",
+    "@radix-ui/react-dialog": "1.0.5",
```

Comment:
These shouldn't be in the root. They should be dependencies of the package that uses them